body_test.go

```diff
@@ -0,0 +1,166 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+var allTheBytes []byte = []byte{1, 3, 7, 15, 31, 63, 127, 255}
```

Comment:
You could use `const` instead of `var` here.

Reply:
When I try:
```go
const allTheBytes []byte = []byte{1, 3, 7, 15, 31, 63, 127, 255}
```
I get:
```
./body_test.go:10: const initializer []byte literal is not a constant
```

I also tried declaring it as a `[8]byte` which also didn't work.  Any idea why it's failing

Reply:
Thanks John, TIL [arrays can't be constants](https://groups.google.com/forum/#!topic/golang-nuts/5aynucvg96I)! You're quite right. :-)